web/src/app/details/Notices.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, ReactNode } from 'react'
+import {
+  Badge,
+  Collapse,
+  Grid,
+  IconButton,
+  makeStyles,
+} from '@material-ui/core'
+import ExpandIcon from '@material-ui/icons/KeyboardArrowDown'
+import CollapseIcon from '@material-ui/icons/KeyboardArrowUp'
+import { Alert, AlertTitle, AlertProps } from '@material-ui/lab'
+import { toTitleCase } from '../util/toTitleCase'
+import { Notice } from '../../schema'
+
+const useStyles = makeStyles({
+  alertAction: {
+    marginRight: 0,
+  },
+  alertMessage: {
+    width: '100%',
+  },
+  gridItem: {
+    padding: '4px 0 4px 0',
+  },
+  lastGridItem: {
+    padding: '4px 0 0 0',
+  },
+  secondGridItem: {
+    padding: '8px 0 4px 0',
+  },
+})
+
+interface NoticesProps {
+  notices: Notice[] // checks against .length are safe as notices is required
+}
+
+export default function Notices(props: NoticesProps): JSX.Element {
+  const classes = useStyles()
+  const [alertsExpanded, setAlertsExpanded] = useState(false)
```

Comment:
Let's be as consistent as we can be with using "notice"
```suggestion
  const [noticesExpanded, setNoticesExpanded] = useState(false)
```